src/style.css

```diff
@@ -0,0 +1,117 @@
+:root {
+  --textCol--color: #060b35;
+}
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+  padding: 32px 16px 16px;
+}
+
+.card_apple {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.stars :last-child {
+  margin-right: 0;
+}
+
+.card_descript {
+  margin: 0 0 4px;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: var(--textCol--color);
+}
+
+.card_rew {
+  height: 16px;
+  margin-bottom: 24px;
+  margin-top: 16px;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.card_code {
+  font-size: 10px;
+  line-height: 14px;
+
+  color: #616070;
+}
+
+.review {
+  color: var(--textCol--color);
+}
+
+.card_price {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.price {
+  width: 32px;
+  height: 18px;
+
+  font-size: 12px;
+  line-height: 18px;
+
+  color: #616070;
+}
+
+.price_value {
+  width: 79px;
+  height: 18px;
+  line-height: 18px;
+
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.card_buy {
+  width: 100%;
+  padding: 12px 0;
+
+  box-sizing: border-box;
+  border: 1px solid transparent;
+  border-radius: 5px;
+
+  text-decoration: none;
+  color: #fff;
+  background-color: #00acdc;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  text-transform: uppercase;
+  font-size: 14px;
+  line-height: 16px;
+  font-weight: 700;
+}
+
+.card_buy:hover {
+  background-color: white;
+  border: 1px solid #00acdc;
```

Comment:
Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover